cmd/crc/cmd/config/view.go

```diff
@@ -31,9 +31,9 @@ func init() {
 
 var configViewCmd = &cobra.Command{
 	Use:   "view",
-	Short: "Display complete crc configurations.",
-	Long: `Retrieves full list of crc configurations. Some of the configuration properties are equivalent
-to the options that you set when you run the 'crc start' command.`,
+	Short: "Display all set crc configuration properties",
```

Comment:
'set' here should be configured perhaps?

in this context, 'set' can be interpreted as 'configurable' and that would convey the wrong message.

Reply:
"Display all configured crc configuration properties" sounds repetitive. Since "config view" only displays the config props that have been assigned values with "config set", I was going with that. I don't in particular like "all set" here, because of https://english.stackexchange.com/questions/21507/is-the-use-of-all-set-exclusive-to-certain-regions.

"Display all assigned crc configuration properties"?

Reply:
it would be a repetition, as this is a verb and a noun form. Somehow, the 'assigned' does not click with me. I'd almost rather would use 'Display all configured properties' . Appreciate feedback.

Reply:
'Display all configured properties' IMO breaks the terminology. Are we manipulating just properties or configuration properties?